### ISSUE TITLE
'sqm -h' fails

### ISSUE THREAD

**Issue by jchodera** [choderalab/ambermini#20]
`conda build ambermini` now fails with
```
+ sqm -h

     Error unknown flag: -h                                                                              
```
which is odd, since this worked fine in AmberTools 14:
```
[LSKI1497:~] choderaj% sqm -h
sqm [-O] -i <input> -o <output>
   -O           Overwrite output file if it exists
   -i <input>   Input file
   -o <output>  Output file
   -h           Show this message
```

**Comment by swails** [comment]
Actually it didn't.  That flag was unique to ambermini.  See af436ce69abba9e3b505cbceef2ca373e5bdd09f

A lot of these kinds of tweaks I made to the actual source code (not the shell script wrappers designed to circumvent the AMBERHOME requirement) got wiped out when I blindly copied AT 15 to ambermini.

I'm going back and adding these modifications back to the main git repo so this doesn't happen again.  I had forgotten how many tweaks I made...
